server/utils/ssr.tsx

```diff
@@ -121,6 +130,11 @@ export const generateMetaComponents = (metaProps: MetaProps) => {
 				<meta key="c5" name="citation_inbook_title" content={collection.title} />,
 				<meta key="c6" name="citation_book_title" content={collection.title} />,
 				<meta key="c7" name="citation_isbn" content={collection.metadata?.isbn} />,
+				<meta
```

Comment:
sorry, I think I got this wrong. This should appear for all types of collections, not just books.